### ISSUE TITLE
Itouch cannot cope with '

### ISSUE THREAD

**Issue by peterverraedt** [irods/irods#7972]
### iRODS Version, OS and Version

* IRODS 4.3.2
* Mysql 8.0 database
* RHEL9

### What did you try to do?

Touching a file containing a `'`, see below. The code snippet speaks for itself.

### Observed behavior (including steps to reproduce, if applicable)

```bash
$ echo test > testfile
$ iput testfile
$ itouch testfile
$ echo test > "test'file"
$ iput test\'file
$ itouch test\'file
Level 0: SYS_NO_GOOD_REPLICA: No good replicas found.

$ ils
/zone/home/project:
  test\'file
  testfile
$ irm testfile
$ irm test\'file
```

**Comment by alanking** [comment]
Duplicate of https://github.com/irods/irods/issues/7169

**Comment by korydraughn** [comment]
This issue was resolved by work on #3902.
